functions/Backup-DbaDbCertificate.ps1

```diff
@@ -240,6 +240,14 @@ function Backup-DbaDbCertificate {
             $InputObject += Get-DbaDbCertificate -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Database $Database -ExcludeDatabase $ExcludeDatabase -Certificate $Certificate
         }
 
+        if ($Certificate) {
+            $missingCerts = $Certificate | ? { $InputObject.Name -notcontains $_ }
```

Comment:
So close! Please avoid using aliases. Also, you'll be excited to know that you can do `Where-Object Name -notcontains $_`, without braces, when comparing just one thing. It's even faster, according to bruce payette.

Reply:
Thanks! I switched it to Where-Object and kept the braces since a caller could pass in multiple values for -Certificate.